pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2313,4 +2312,15 @@ self: super: {
   # multiple bounds too strict
   co-log-polysemy = doJailbreak super.co-log-polysemy;
   co-log-polysemy-formatting = doJailbreak super.co-log-polysemy-formatting;
+
+  # 2022-12-02: Needs newer postgrest package
```

Can you link this issue in the comment: https://github.com/PostgREST/postgrest/issues/2275? This getting fixed would allow us to stop tracking the source manually.

Also please explain why `doJailbreak` and `doCheck` are needed – for the former opening an issue with upstream usually makes sense as well.

---

`doJailbreak` is needed for dependency issues:

```bash
Setup: Encountered missing or private dependencies:
       > HTTP >=4000.3.7 && <4000.4,
       > lens-aeson >=1.0.1 && <1.2,
       > optparse-applicative >=0.13 && <0.17
```

Meanwhile, `doCheck` is actually disabled for all pgSQL packages as they require a running server.

---

Then I'd do:

```suggestion
  # 2022-12-02: Needs newer postgrest package
  # 2022-12-02: Hackage release lags behind actual releases: https://github.com/PostgREST/postgrest/issues/2275
  # 2022-12-02: Too strict bounds: https://github.com/PostgREST/postgrest/issues/2580
  # 2022-12-02: Tests require running postresql server
```

Note that `dontCheck` could be removed if #168932 is used, but this depends on the test suite accepting unix sockets as a pg connection. In any case it's not required.

---

Done.